### PR TITLE
update/redbot images

### PR DESCRIPTION
**Pull request by gOOvER**
# Description

- update egg to PTDL_v2
- switch image to ghcr
- update installimage to phyton 3.10

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### CODE REVIEW

**Comment by QuintenQVD0** [comment]
I agree with changing the game image to the new maintained one

But the install image must be still python 3.8 as that is what the use for the github workflows.
It also can not get the bot installed with python 3.10 as the installer image, works fine with the new game images and python:3.8-slim image

```error
ERROR: Ignored the following versions that require a different python version: 3.0.0 Requires-Python >=3.6.2,<3.8; 3.0.0b16 Requires-Python >=3.6,<3.7; 3.0.0b17 Requires-Python >=3.6,<3.7; 3.0.0b18 Requires-Python >=3.6,<3.7; 3.0.0b19 Requires-Python >=3.6,<3.7; 3.0.0b20 Requires-Python >=3.6,<3.8; 3.0.0b21 Requires-Python >=3.6.2,<3.8; 3.0.0rc1 Requires-Python >=3.6.2,<3.8; 3.0.0rc1.post1 Requires-Python >=3.6.2,<3.8; 3.0.0rc2 Requires-Python >=3.6.2,<3.8; 3.0.0rc3 Requires-Python >=3.6.6,<3.8; 3.0.0rc3.post1 Requires-Python >=3.6.2,<3.8; 3.0.1 Requires-Python >=3.6.2,<3.8; 3.0.2 Requires-Python >=3.6.2,<3.8; 3.4.1 Requires-Python <3.9,>=3.8.1; 3.4.10 Requires-Python <3.9,>=3.8.1; 3.4.11 Requires-Python <3.9,>=3.8.1; 3.4.12 Requires-Python <3.9,>=3.8.1; 3.4.13 Requires-Python <3.10,>=3.8.1; 3.4.14 Requires-Python <3.10,>=3.8.1; 3.4.15 Requires-Python <3.10,>=3.8.1; 3.4.16 Requires-Python <3.10,>=3.8.1; 3.4.17 Requires-Python <3.10,>=3.8.1; 3.4.18 Requires-Python <3.10,>=3.8.1; 3.4.2 Requires-Python <3.9,>=3.8.1; 3.4.3 Requires-Python <3.9,>=3.8.1; 3.4.4 Requires-Python <3.9,>=3.8.1; 3.4.5 Requires-Python <3.9,>=3.8.1; 3.4.6 Requires-Python <3.9,>=3.8.1; 3.4.7 Requires-Python <3.9,>=3.8.1; 3.4.8 Requires-Python <3.9,>=3.8.1; 3.4.9 Requires-Python <3.9,>=3.8.1
ERROR: Could not find a version that satisfies the requirement Red-DiscordBot (from versions: 3.1.0, 3.1.1, 3.1.2, 3.1.3, 3.1.4, 3.1.5, 3.1.6, 3.1.7, 3.1.8, 3.1.9, 3.2.0, 3.2.1, 3.2.2, 3.2.3, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.3.4, 3.3.5, 3.3.6, 3.3.7, 3.3.8, 3.3.9, 3.3.10, 3.3.11, 3.3.12, 3.4.0)
ERROR: No matching distribution found for Red-DiscordBot
```

**Comment by gOOvER** [comment]
They recommended Ubuntu 22.04 as Distro and Ubuntu 22.04 have python 3.10 as default. So it should run

**Comment by QuintenQVD0** [comment]
If I try to install with the install images as python 3.10 it fails. If I use 3.8 then it installs fine

File: bots/discord/redbot/egg-red.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
...
-            "container": "python:3.8-slim",
+            "container": "python:3.10-slim",
```
Comment (QuintenQVD0) [inline bots/discord/redbot/egg-red.json]: Let just keep this as `python:3.8-slim` as the mention 3.8.1 or 3.9 is supported

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

**Comment by QuintenQVD0** [comment]
Thank you, Looks good to me!

**Review by QuintenQVD0 — APPROVED** [review]

**Pull request merged**